kitchen.recipes-config.yml

```diff
@@ -291,3 +291,39 @@ suites:
         - network_interfaces_configuration_script_created
         - network_interfaces_configured
         - multiple_network_interfaces_can_ping_gateway
+  - name: custom_slurm_settings_file_via_s3
+    run_list:
+      - recipe[aws-parallelcluster::add_dependencies]
+      - recipe[aws-parallelcluster-slurm::retrieve_remote_custom_settings_file]
+    verifier:
+      controls:
+        - custom_settings_file_retrieved
+    attributes:
+      dependencies:
+        - recipe:aws-parallelcluster-install::directories
+        - recipe:aws-parallelcluster-install::python
+      cluster:
+        config:
+          Scheduling:
+            SchedulerSettings:
+              # here we are just interested in testing that we can retrieve a file from S3 through S3 protocol
+              CustomSlurmSettingsIncludeFile: 's3://us-east-1-aws-parallelcluster/templates/1-click/serverless-database.yaml'
+              LocalPath: '/tmp/custom_slurm_settings_include_slurm.conf'
+  - name: custom_slurm_settings_file_via_https
```

Comment:
Can we put a small comment here briefly describing the scope of the test?

Reply:
Done